packages/typespec-test/test/anomalyDetector/generated/typespec-ts/src/rest/responses.ts

```diff
@@ -83,7 +83,7 @@ export interface GetMultivariateBatchDetectionResultDefaultHeaders {
 export interface GetMultivariateBatchDetectionResultDefaultResponse
   extends HttpResponse {
   status: string;
-  body: MultivariateResponseErrorOutput;
+  body: MultivariateErrorResponseOutput;
```

Comment:
This is introduced by compiler change https://github.com/microsoft/typespec/pull/2945.

So openapi.json has similar changes.